db/tikv/coprocessor.go

```diff
@@ -112,5 +170,18 @@ func (db *coprocessor) Insert(ctx context.Context, table string, key string, val
 }
 
 func (db *coprocessor) Delete(ctx context.Context, table string, key string) error {
-	return errors.New("unsupported type of delete in coprocessor")
+	// encode key
```

Comment:
Do we need to first query for existence before actually deletion, to simulate TiDB's access pattern here? @siddontang

Reply:
In YCSB, there is no delete in core-workload, so the delete function will never be used.

Reply:
I prefer supporting it, it is easy. And we also need to use delete in shell mode.